README.md

```diff
@@ -26,6 +26,34 @@ Returns output that looks like:
 
     Kubernetes master is running at https://146.148.80.79
 
+In our configuartion, we also rely on persistent disks created by the Google Cloud Platform for storage. You may provision a disk on the Google Cloud Shell using the following command:
+	
+	gcloud compute disk create your-disk-name-here --size 10GiB
```

Comment:
Also nit: can you use spaces here instead of tab characters?

In your Sublime config:

    "translate_tabs_to_spaces": true,